### PR TITLE
lab 8 directives - controllerAs updates

### PR DESCRIPTION
**Pull request by hartstinger**
#68 controllerAs

### CODE REVIEW

File: client/test/unit/app/controllers.spec.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ describe('App', function() {
     describe('MainCtrl', function() {
       beforeEach(inject(function($rootScope, $controller) {
         scope = $rootScope.$new();
-        controller = $controller("MainCtrl", { 
-          $scope: scope 
+        controller = $controller("MainCtrl", {
+          $scope: scope
```
Comment (tmartensen) [inline client/test/unit/app/controllers.spec.js:23]: Maybe forgot this one?

**Comment by tmartensen** [comment]
:+1:

File: client/test/unit/app/controllers.spec.js (hunk 1 above)
Comment (hartstinger) [inline client/test/unit/app/controllers.spec.js:23]: MainCtrl controller injects $scope for $watch so I have to leave this one here.

Comment (tmartensen) [inline client/test/unit/app/controllers.spec.js:23]: Oh yeah...forgot. My bad.

**Pull request merged**